### ISSUE TITLE
Starting up the proxy

### ISSUE THREAD

**Issue by RoelofJ** [Esri/resource-proxy#176]
Hi,

I installed the DotNet Proxy and I want to use it in both my development and testing environment. In development it works as expected, but on testing it doesn't. I copied all files from the DotNet Proxy to my testing environment, following this guide: https://developers.arcgis.com/javascript/jshelp/ags_proxy.html

When I try to reach anything through this proxy when it's on the testing environment I get an error like: 
{error: {code: 500,message:"Unable to connect to the remote server http:// services.arcgisonline.com/ArcGIS/rest/services/?f=pjson"}}

I'm sure I'm missing something, but I can't figure out what it is. Can you help me?

**Comment by RoelofJ** [comment]
I had tested it with the same identity. Somehow we did resolve the problem now and it had to do with the identity given in the application pool not having access to the internet. We now put in another identity there with the access and that resolved the problem. I still don't know why it did work earlier with the 'standard' identity the IIS put in, but it works :) Thanks for all the help! :)

**Comment by jgravois** [comment]
glad to hear i was able to point you in the right direction @roelofj.  thanks for taking the time to articulate the resolution.

**Comment by apurvdanke** [comment]
I am having exactly the same problem. I changed the app pool identity to local system, local service, network service and also custom account, which is my own server administrator account (it is a domain account which is the admin for this server). I created a new app pool for the DotNet proxy and did the changes in that one, not the default app pool. I kept the type as integrated. Suggestions welcome.

Regards,
Apurv
